include/materials/NuclearMaterial.h

```diff
@@ -64,6 +64,13 @@ class NuclearMaterial : public GenericConstantMaterial
 
   const VariableValue & _temperature;
 
+  // Number of neutron groups
+  unsigned int _num_groups;
```

Comment:
Will this cause issues if a user tries to use more than 255 groups? I don't think it's common, but possibly worth considering for future-proofing.

Reply:
I think unsigned int goes up to 65k so it should be okay